sdk/internal/testframework/recording.go

```diff
@@ -0,0 +1,420 @@
+// +build go1.13
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package testframework
```

Comment:
I've been struggling to come up with a better name.  Do you think there will be content in here other than recordings?

Reply:
> Do you think there will be content in here other than recordings? It it internal code, so not super-critical.

Probably not.. Were you thinking something like `recordedtests` or `recordedtestfixture`?

Reply:
I was think a bit more brief, like `recording`.  So code would look like `recording.NewRecording()`.  This would also facilitate removal of any `Recording` prefix on type names, e.g. `RecordingSanitizer` would simply become `Sanitizer` to avoid stuttering (i.e. avoid `recording.RecordingSanitizer`).

Reply:
I like it!